tutorials/iot/ubuntu-kiosk/x11-kiosk.md

```diff
@@ -249,9 +245,14 @@ apps:
 parts:
```

Comment:
I think this *snapcraft.yaml* is missing `base: core18`, and `version:` will need to become a string.

Reply:
Indeed